modules/op-console-statistics.adoc

```diff
@@ -7,7 +7,10 @@
 
 You can view statistics related to execution of pipelines in the web console.
 
-To view the statistic information, you must first enable the {pipelines-shortname} console plugin.
+To view the statistic information, you must complete the following steps:
+
+* Install {tekton-results}
```

Comment:
Do we want to add something like: For more infromation, see _Using Tekton Results for OpenShift Pipelines observability_ in the _Additional resources_ section.

Give folks some sense of where to go to install it.